Calculator/Calculator.cpp

```diff
@@ -44,13 +44,10 @@ vector<string> str_to_strvec(string &equ_str) {
             if (i == 0 || i == (equ_str.length() - 1)) {        //If character is the first or last charcter, throw an error
                 throw(runtime_error("Operator is missing an operand."));
             }
-            else if (char_is_num(equ_str[i - 1]) == true && char_is_num(equ_str[i + 1]) == true) { //character is preceded and followed by a number
-                equ_vec.push_back(character);                   //add the string version to the end of equ_vec.
+            else if ((char_is_num(equ_str[i - 1]) == true || equ_str[i - 1] == ')') && (char_is_num(equ_str[i + 1]) == true || equ_str[i + 1] == '-' || equ_str[i + 1] == '(')) {

```

Comment:
this if statement is handling a lot of conditions; we should maybe try to refactor this somehow